### ISSUE TITLE
Not really working

### ISSUE THREAD

**Issue by krtschmr** [bitpay/spree-bitpay#18]
Installation failed, field `pem` is too short. changed to `:text`

connected to TestNet works, Livenet happens nothing and development failes (redirect no matching route)

As for payment method choice

![image](https://cloud.githubusercontent.com/assets/13366714/17524083/3d0a60da-5e87-11e6-910c-4c762a5f0b67.png)

notihng else shows up.

Spree 3.1.0

is that normal behavior?

**Comment by philosodad** [comment]
The last version tested against was 3.0.1, and development has been discontinued, so I would not necessarily expect this to run with Spree 3.1.x. 

It's possible that livenet is failing because of a missed configuration, IE, not using the right environment, or it's possible that bitpay has changed the location of an endpoint.

**Comment by krtschmr** [comment]
yeah, so then i stick to my own solution using the bitcoin_payable gem 
https://github.com/Sailias/bitcoin_payable
